corehq/apps/app_manager/views/releases.py

```diff
@@ -403,11 +411,20 @@ def revert_to_copy(request, domain, app_id):
         copy_build_comment_template = _("Reverted to version {old_version}")
 
     try:
+        user_id = request.couch_user.get_id
         copy = app.make_build(
             comment=copy_build_comment_template.format(**copy_build_comment_params),
-            user_id=request.couch_user.get_id,
+            user_id=user_id,
         )
         copy.save(increment_version=False)
+        if toggles.APPLICATION_RELEASE_LOGS.enabled(domain):
```

Comment:
I wonder if we should always create these, and just not display them unless the flag is on. Given that the feature flag is on the product path, it'd be nice if whenever projects turn this on, and when the flag is eventually released (fingers crossed), the data is already available. @pxwxnvermx @mkangia what do you think?

Reply:
We did discuss about this and the argument that I had in favor of keeping it this way was that since this is a new feature flag, it should not do anything as a part of the core HQ for now. 
I want to avoid the situation when there is an issue in app release feature because something goes off here.
Till we have more confidence in it and have enabled it for few projects on production, we can then ask product to do a pass on it and then enable it for all projects at least from the logging perspective like you said, and then in future, SaSS can make it available in view as well whenever they are ready.